docs/advanced-topics/checksum-support.rst

```diff
@@ -37,6 +50,22 @@ Descriptions for the two endpoints of note are as follows:
 
 Just like the file endpoints, the id parameter used in the endpoints above can be found on an entry's public page; underneath the Info tab, look for the bolded words **TRS**.
 
+To verify a checksum as reported by the Dockstore API matches what you download from the Docker registry first find the checksum
+and image path using one of the above methods for the image you would like to verify. Then download the image using the
+Docker CLI client.
+
+::
+
+    docker pull quay.io/briandoconnor/dockstore-tool-md5sum:1.0.4
+
+When the download has completed a Digest is provided in the terminal output. This should match the checksum provided
+by the Dockstore API.
+
+Verifying the image checksum can give you better guarantees the image has not changed since it was published to Dockstore.
```

Comment:
"...image has not changed since it was published to Dockstore... " -- nit-picky and optional, but I interpret "it" as the image, but I think you mean since the workflow was published to Dockstore.

Or if you do mean the image, it's not published to Dockstore, it's published to DockerHub, Quay, etc.

Reply:
Will clarify that I meant workflow